esphome/components/dallas/sensor.py

```diff
@@ -24,7 +24,7 @@
     ).extend(
         {
             cv.GenerateID(CONF_DALLAS_ID): cv.use_id(DallasComponent),
-            cv.Optional(CONF_ADDRESS): cv.hex_int,
+            cv.Optional(CONF_ADDRESS): cv_dallas_address,
```

Comment:
We have a validator for this already =)
```suggestion
            cv.Optional(CONF_ADDRESS): cv.hex_uint64_t,
```

Reply:
Not familiar with the magic yet, thanks :)